File: Tamrin12shahrivar/ProjectFront/GemAngularApp/src/app/models/gem.ts


export class Gem {
  gemId: string | null;
  data: string | null;
  weight: number|null;
  cutie: number|null;
  price: number|null;
  status: string | null;

  constructor(gemId: string | null = null, data: string | null = null, weight: number | null = null, cutie: number | null = null, price: number | null = null, status: string | null = null) {
    this.gemId = gemId,
      this.data = data,
      this.weight = weight,
      this.cutie = cutie,
      this.price = price,
      this.status = status
  }
}

File: Tamrin12shahrivar/Model/Gem.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Tamrin12shahrivar.Model
{
    public class Gem
    {
        public Guid GemId { get; set; }

        public DateTime Data { get; set; }

        [Column(TypeName = "decimal(18,2)")]
        public decimal Weight { get; set; }

        public int cutie { get; set; }

        public long Price { get; set; }



        public string Status { get; set; }

    }
}
